scvi/model/_scvi.py

```diff
@@ -30,7 +30,7 @@
 
 _SCVI_LATENT_QZM = "_scvi_latent_qzm"
 _SCVI_LATENT_QZV = "_scvi_latent_qzv"
-
+_SCVI_LATENT_MODE = "posterior_parameters"
```

Comment:
nit-ish: We might choose to have other latent mode types for scvi, so I would name this something more to the effect of "posterior_parameters" like `POSTERIOR_PARAMS_LATENT_MODE` or something rather than to the fact that this is scvi's latent mode because in theory scvi might have different types of latent mode if we want it. What do you think?

Reply:
I left this as a TODO